test/Import/template-specialization/test.cpp

```diff
@@ -1,4 +1,5 @@
 // RUN: clang-import-test -import %S/Inputs/T.cpp -expression %s
+// XFAIL: *
```

Comment:
This test is failing in llvm.org/master too, see:
https://github.com/llvm-mirror/clang/blob/master/test/Import/template-specialization/test.cpp#L2